src/dune_engine/print_diff.mli

```diff
@@ -7,3 +7,16 @@ val print :
   -> Path.t
   -> Path.t
   -> _ Fiber.t
```

Comment:
This function should be `Diff.or_raise : t -> _ Fiber.t` I suppose.

Reply:
Done (I chose to return `User_message.t` - this is ignored anyway)